state/txindex/kv/utils.go

```diff
@@ -1,4 +1,4 @@
-package common
+package kv
 
 // IntInSlice returns true if a is found in the list.
 func IntInSlice(a int, list []int) bool {
```

Comment:
Do you think it makes sense to a) make the func private, because it's not core API of this package b) move it closer to where it's used ?

Reply:
YES!

Reply:
went with A